.travis.yml

```diff
@@ -54,47 +44,38 @@ jobs:
   allow_failures:
     - php: nightly
     - env: DB=pgsql POSTGRESQL_VERSION=11.0
-
-  exclude:
-    - php: 7.1
-      env: DB=sqlite
-    - php: 7.1
-      env: DB=mysql
-    - php: 7.1
-      env: DB=mysqli
+    - php: 7.3
+      env: DB=ibm_db2
```

Comment:
It should be just excluded from the matrix.

Reply:
It may start working at any time, though, once a newer `ibm_db2` PECL package is available; shouldn't we keep it as an allowed failure in the meantime?